app/models/video.rb

```diff
@@ -0,0 +1,20 @@
+class Video < ApplicationRecord
+  has_many :rentals, :dependent => :destroy
+  has_many :customers, through: :rentals
+
+  validates :title, presence: true
+  validates :overview, presence: true
+  validates :release_date, presence: true
+  validates :total_inventory, presence: true
+  validates :available_inventory, presence: true
+
```

Comment:
It would also be good to have methods to handle the business logic of checking in and checking out videos.